grammar.js

```diff
@@ -216,19 +216,26 @@ module.exports = grammar({
         ")"
       ),
 
+    _qualified_call: $ => alias($.qualified_call, $.call),
+
     qualified_call: $ => seq(field("name", $.identifier), $.arguments),
 
     call: $ =>
       prec(
         PREC.CALL,
         choice(
           seq(
-            field("name", choice($.identifier, $.dot_call, $.qualified_call)),
-            choice($._bare_arguments, $.arguments),
+            field("name", choice($.identifier, $.dot_call, $._qualified_call)),
+            seq(
+              choice(
+                alias($.bare_arguments, $.arguments),
```

Comment:
could you leave the argument change out of this PR

Reply:
done